Hello/about.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Об авторе</title>
+  </head>
+  <body>
+    <h1>Приветствую!</h1>
+    <h3>Почему я выбрала Front-end?</h3>
+    <p>
+      Я люблю творчество и логические задачки. Иногда нестандартно нахожу
+      решение. Желание заняться программированием возникло с появлением
+      удаленного формата работы, выходом в декретный отпуск.
+      <br />Немаловажна привлекающая заработная плата разработчиков &#x1F63A;.
+      Ну а наиболее творческое направление я увидела у
+      <em>Front-end</em> разработчиков. Сюда и направилась.
+    </p>
+    <h3>Напишу немного о себе</h3>
+    <p>
+      Родилась я 29 лет назад. Школа, медицинский университет, интернатура,
+      работа. На данный момнет я работаю в
+      <del>поликлинике</del>
+      <abbr title="Гомельская областная инфекционная клиническая больница"
+        >ГОИКБ</abbr
+      >
+      врачом-инфекционистом <ins>приемного отделения</ins>. Почему я выделила
+      отделение? Да потому, что работа в приемнике - это другая жизнь, это
+      "мотор" больницы, это адреналин и многозадачность, разнообразие и
+      сложности... Могу продолжать долго описывать плюсы и минусы этой
+      работы.<br />
+      В свободное время я пишу картины для себя и на заказ. С мужем создаем
+      интерьерный свет. И подарки знакомым, близким не обходятся без моего
+      творческого вмешательства. <br />Из рассказа о себе бОльшая часть
+      получилась про работу. Это показывает 1) как я соскучилась по ней,
+      находясь в декрете; 2)и как я люблю работать. Часто я много на себя беру.
+      Плохо организованное время и стратегия решения задачи увеличивают время
+      выполнения. <mark>Но я над этим работаю!</mark>
+    </p>
+    <p>
+      Я поставила себе цель за год обучиться программированию и начать работать
+      в этой сфере. Со помощью команды школы
+      <strong
+        ><a href="https://intellectfox.by/" target="_blank"
+          >Intellect Fox</a
+        ></strong
+      >
+      я должна добиться успехов!
+    </p>
+    <p>
+      Мои работы я выкладываю в <sub>irina.pi</sub>инстаграмм-аккаунт<sup
+        >irina.pi</sup
+      >
+      и я вас приглашаю с ними
+      <a
+        href="https://www.instagram.com/stories/highlights/17856019180760371/"
+        target="_blank"
+        >познакомиться!</a
+      >
+    </p>
+    <small
+      >Моя рабочая почта:<a href="mailto:rabotairinabocharova@yandex.by"
+        >rabotairinabocharova@yandex.by</a
+      ></small
+    >
+    <hr />
+    <a href="../index.html">Home</a>
+  </body>
+</html>
```

Comment:
Used 2 types of headers only.
No quotes, link to github.

Reply:
I corrected already.